<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>300</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>413</width>
    <height>300</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="playlistLinkBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Playlist Link</string>
    </property>
   </widget>
   <widget class="QComboBox" name="formatComboBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>FLAC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>WAV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MP3 (Not Recomended)</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="downloadButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="spotifyID">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>202</x>
      <y>30</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="spotifySecret">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>202</x>
      <y>60</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="downloadBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>391</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Client</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Secret</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="savePathBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Save Path</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>413</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_Keys"/>
    <addaction name="actionImport_Keys"/>
    <addaction name="actionRemove_Keys"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRemove_Keys">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Remove Keys</string>
   </property>
  </action>
  <action name="actionImport_Keys">
   <property name="text">
    <string>Load Keys</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionExport_Keys">
   <property name="text">
    <string>Export Keys</string>
   </property>
  </action>
  <action name="actionSave_Keys">
   <property name="text">
    <string>Save Keys</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
